### DISCUSSION TITLE
car status ?

### DISCUSSION THREAD

**Discussion by herbert131** [goecharger/go-eCharger-API-v2#215] [Q&A]
what is the difference between car = 3 and car = 4 ?
So far when I stopped charging car status was set to 3 now I realized it is set to 4 even if battery is not full

**Comment by erxbout** [comment]
As there is no option to detect the SoC of the connected car the state of charging is set to 4 (Complete) when stopping the charge..
The state 3 (WaitCar) is set before the charging starts (you plug it in, then the car checks the voltages and stuff or is configured to charge on a timer setting)
Also if you do not enable charging on the charger and a car is connected its in the wait car state

**Reply by erxbout** [reply]
more information here https://github.com/goecharger/go-eCharger-API-v2/discussions/206
